.circleci/config.yml

```diff
@@ -237,12 +239,41 @@ jobs:
           # --ignore ref: https://github.com/crossbeam-rs/crossbeam/issues/401
           # hyper/reqwest/tokio -> tokio-threadpool -> crossbeam-deque -> crossbeam-epoch -> memoffset
           cargo audit --ignore RUSTSEC-2019-0011
+          mkdir -m 755 bin
+          mkdir -m 755 bin/config
+          cargo build --release
+          cp config/* bin/config
+          cp target/release/fxa_email_send bin
+          cp target/release/fxa_email_queues bin
+          cargo clean
       - store_artifacts:
           path: fxa-auth-db-mysql.log
       - setup_remote_docker
       - run: |
-          ../../.circleci/build.sh fxa-email-service
-          ../../.circleci/deploy.sh fxa-email-service
+          ../../.circleci/tag.sh fxa-email-service
```

Comment:
How come both this and `fxa-email-service-tag` do the release build and tag stuff? Does it need to be here, or would it be sufficient for this job to stop after `cargo audit`?

Reply:
Mainly to match prior behavior where it was build and deployed in both places. I'm not sure why it was setup for that either.

Reply:
Nvm, I just remembered why. It's so that `feature.xxx` and `dockerpush.yyy` branches pushed to dockerhub.